src/index.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import { Provider } from 'react-redux';
-import { createStore } from 'redux';
+// import { createStore } from 'redux';
```

Comment:
Always remove commented code, it is better to use git history.

Reply:
Fixed the issue.